### ISSUE TITLE
Next Week Plan (04-09-21)

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#7]
Plan for next week
- ~Understand CircuitQNN from the qiskit machine learning~
- ~Generate an easier dataset with two separable circles~ 
- ~State vector simulator~
- ~Regularization term, (separable case, it is okay to remove~
- search for optimal hyperparameters 
-  Loss convergence to zero.

Meeting summary
- Discuss the hyperparameters
- How paper seperat the data, and validate accuracy
- Regularization soft margin, hard margin
- Pytorch, qiskit machinlearnign module
- Import *
- pdf files for near term quantum machine learning
